web/.env.example

```
@@ -14,24 +14,35 @@ DATABASE_URL=
-RIVET_RUNNER_VERSION=
+# Cloud VM database. Local dev derives DATABASE_URL from CMUX_PORT. Staging/production use the
+# Vercel Marketplace AWS Aurora PostgreSQL OIDC/RDS IAM env set below.
+CMUX_DB_DRIVER=
+AWS_ROLE_ARN=
+AWS_REGION=
+PGHOST=
+PGPORT=
+PGUSER=
+PGDATABASE=
+CMUX_DB_POOL_MAX=5
+CMUX_DB_SSL_REJECT_UNAUTHORIZED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CMUX_DB_SSL_REJECT_UNAUTHORIZED key should go before the PGDATABASE key

(UnorderedKey)